api/deploy/kubernetes/nfs/csi-provisioner-rbac-crb.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: "{{ .ServiceAccount }}-role"
```

Comment:
Because we allow the user to pick a name for the service account. I think it is nice to have names of roles easily related with service accounts. If you prefer a different naming convention, can you give a suggestion?

Reply:
i dont have any preference but just wanted to know why this flexibility for users here but for Role/ClusterRole names.